instrumentation/opentelemetry-api/opentelemetry-api-1.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/opentelemetryapi/trace/ApplicationSpan.java

```diff
@@ -100,6 +100,20 @@ public Span addEvent(
     return this;
   }
 
+  @Override
+  @CanIgnoreReturnValue
+  public Span addLink(SpanContext spanContext) {
+    agentSpan.addLink(Bridging.toAgent(spanContext));
+    return this;
+  }
+
+  @Override
+  @CanIgnoreReturnValue
+  public Span addLink(SpanContext spanContext, Attributes attributes) {
+    agentSpan.addLink(Bridging.toAgent(spanContext), Bridging.toAgent(attributes));
+    return this;
+  }
+
```

Comment:
this deviates a bit from our usual approach of adding bridging for new methods in a subclass, but here it is much easier to do it the way it is so I think it is fine.